app/services/access_token_verifier.rb

```diff
@@ -10,7 +10,12 @@ def initialize(http_authorization_header)
   end
 
   def submit
-    FormResponse.new(success: valid?, errors: errors)
+    FormResponse.new(
+      success: valid?, errors: errors, extra: {
+        client_id: identity&.service_provider,
```

Comment:
can we update the `openid_connect_bearer_token` event params and its docs too please? 🙏